core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -291,24 +295,26 @@ var ExplorationEditorMainTab = function() {
   };
 
   // Rules are zero-indexed; 'default' denotes the default outcome.
+  // 'pop' denotes the currently opened one.
   this.getResponseEditor = function(responseNum) {
-    var headerElem;
-    if (responseNum === 'default') {
-      headerElem = defaultResponseTab;
-    } else {
-      headerElem = responseTab.get(
-        responseNum);
-    }
-
-    responseBody(responseNum).isPresent().then(function(isVisible) {
-      if (!isVisible) {
-        expect(headerElem.isDisplayed()).toBe(true);
-        waitFor.elementToBeClickable(
-          headerElem, 'Response Editor header is not clickable');
-        headerElem.click();
+    if (responseNum !== 'pop') {
+      var headerElem;
+      if (responseNum === 'default') {
+        headerElem = defaultResponseTab;
+      } else {
+        headerElem = responseTab.get(
+          responseNum);
       }
-    });
 
+      responseBody(responseNum).isPresent().then(function(isVisible) {
+        if (!isVisible) {
+          expect(headerElem.isDisplayed()).toBe(true);
```

Comment:
For `responseNum === 'pop'` what will be `headerElem`?

Reply:
These lines will not be executed.  It jumps to the return statement directly.

Reply:
I'm not sure whether `headerElem` will be defined for `responseNum === 'pop'`?

Reply:
No, it would not.
But I have added one line to set `headerElem` to `addResponseHeader` and wait for it to be displayed.